apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -249,7 +267,11 @@ export default class AnimationPickerBody extends React.Component {
                     {`${msg.animationPicker_allCategories()} > `}
                   </span>
                 )}
-                <span>{msg[`animationCategory_${categoryQuery}`]()}</span>
+                <span>
+                  {categoryQuery !== 'level_animations'
+                    ? msg[`animationCategory_${categoryQuery}`]()
+                    : 'Costumes'}
```

Comment:
As stated in comment above, 'Costumes' is a more accurate label for level-specific animations that are not backgrounds.